GaugeWebsite/server.js

```diff
@@ -27,7 +28,7 @@ app.use(bodyParser.urlencoded({
 app.use(express.static(publicDir));
 
 app.get('/data/data.json', function (req, res) {
-  _Db.collection('data').find({ level: { $exists: true } }, { _id: true, level: true }).sort({ _id: 1 }).limit(2000, function (err, results) {
+  _Db.collection('data').find({ level: { $gt: 0 } }, { _id: true, level: true }).sort({ _id: 1 }).limit(2000, function (err, results) {
```

Comment:
Ooh and fixed this, $exists: true returns true when the level is blank/zero. switched to Greater than 0 to eliminate gaps in the graph if we get a bad reading/parse